azext_iot/tests/digitaltwins/test_dt_import_jobs_int.py

```diff
@@ -0,0 +1,177 @@
+        sleep(60)
+
+        # Upload import data files to storage account
+        valid_import_data_file = "./import_data/bulk-models-twins-relationships-valid.ndjson"
+        self.cmd(
+            "storage blob upload --connection-string '{}' --container-name '{}' --file '{}' --overwrite".format(
+                self.storage_cstring, self.storage_container, valid_import_data_file
+            )
+        )
+        invalid_import_data_file = "./import_data/bulk-models-twins-relationships-invalid.ndjson"
+        self.cmd(
+            "storage blob upload --connection-string '{}' --container-name '{}' --file '{}' --overwrite".format(
+                self.storage_cstring, self.storage_container, invalid_import_data_file
+            )
+        )
+
+        # Create import job for valid import data
+        valid_import_job_output_filename = "{}_valid_import_job_output.txt".format(instance_name)
+        create_valid_import_job_output = self.cmd(
+            "dt job import create -n '{}' -g '{}' --df '{}' --ibc '{}' --isa '{}' --of '{}'".format(
+                instance_name, self.rg, valid_import_data_file.rsplit('/', maxsplit=1)[-1],
+                self.storage_container, self.storage_account_name, valid_import_job_output_filename
+            )
+        ).get_output_in_json()
+
+        assert_import_job_creation(create_valid_import_job_output)
+        valid_import_job_id = create_valid_import_job_output["id"]
+
+        # Show import job
+        show_import_job_output = self.cmd(
+            "dt job import show -n '{}' -g '{}' -j '{}'".format(instance_name, self.rg, valid_import_job_id)
+        ).get_output_in_json()
+        assert(show_import_job_output["id"] == valid_import_job_id)
+
+        # Create import job for invalid import data
+        invalid_import_job_output_filename = "{}_invalid_import_job_output.txt".format(instance_name)
+        create_invalid_import_job_output = self.cmd(
+            "dt job import create -n '{}' -g '{}' --df '{}' --ibc '{}' --isa '{}' --of '{}'".format(
+                instance_name, self.rg, invalid_import_data_file.rsplit('/', maxsplit=1)[-1],
+                self.storage_container, self.storage_account_name, invalid_import_job_output_filename
+            )
+        ).get_output_in_json()
+
+        assert_import_job_creation(create_invalid_import_job_output)
+        invalid_import_job_id = create_invalid_import_job_output["id"]
+
+        # List import jobs
+        list_import_jobs_output = self.cmd(
+            "dt job import list -n '{}' -g '{}'".format(instance_name, self.rg)
+        ).get_output_in_json()
+        assert len(list_import_jobs_output) == 2
+        import_job_ids = [valid_import_job_id, invalid_import_job_id]
+        assert list_import_jobs_output[0]["id"] in import_job_ids
+        assert list_import_jobs_output[1]["id"] in import_job_ids
+
+        # Poll to ensure desired status of import jobs
+        num_tries = 0
+        max_tries = 5
```

Comment:
might want to make this and the sleep interval into constants at the beginning of the file for easier manipulation